event/event.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/ipfs/go-cid"
 	"github.com/libp2p/go-libp2p/core/peer"
 
+	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
+	"github.com/sourcenetwork/immutable"
```

Comment:
nitpick: These imports arent correctly formatted, the linter is just confused.

`immutable` is external to this repo and belongs in the same block as `github.com/libp2p/go-libp2p/core/peer`.  `acpIdentity` should be in the same block as `client`